web/themes/new_weather_theme/assets/sass/components/forecast-details-toggle.scss

```diff
@@ -0,0 +1,13 @@
+@use "uswds-core" as *;
+
+.wx-forecast-details-toggle ~ * {
+    display: none;
+}
+
+.wx-forecast-details-toggle[data-selected="charts"] ~ *:not(wx-hourly-table) {
+    display: initial;
+}
+
+.wx-forecast-details-toggle[data-selected="table"] ~ wx-hourly-table {
+    display: initial;
+}
```

Comment:
This works great for all but the QPF/precipitation amounts because the chart and table are combined in one include, so when you switch to "chart" it shows both the table and the chart and when you chose "table" it hides both. Probably need to pull chart into it's own partial.

Reply:
Adding here that similar thing happens to the heading "Hourly forecast" because that one is inside the table partial, and I think we should probably pull it out into the `daily-forecast-list-item`

Reply:
Oops. That QPF chart and table being in the same template was my fault. 🤦‍♂️